spec/mailers/admin_mailer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
```

Comment:
There must have been some awkwardness in the merging. It passed because the method its referencing also got merged back in. I've removed both the file and the method.